Experimental/MultiplayerUseCases/Packages/manifest.json

```diff
@@ -10,7 +10,7 @@
     "com.unity.learn.iet-framework.authoring": "1.2.2",
     "com.unity.multiplayer.samples.coop": "https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop.git?path=/Packages/com.unity.multiplayer.samples.coop#v2.4.0",
     "com.unity.multiplayer.tools": "1.1.1",
-    "com.unity.netcode.gameobjects": "1.7.1",
```

Comment:
Let's also update the project's readme's badge to increment that to 1.8.1 as well.

Reply:
Interesting, I did not even notice before that we have this badge! We should change it on the main page to 1.8.1 as well then. I am gonna change the local README of Multiplayer Use Cases